gtpv2/message/change-notification-res_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019-2022 go-gtp authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package message_test
+
+import (
+	"testing"
+
+	"github.com/wmnsk/go-gtp/gtpv2/ie"
+	"github.com/wmnsk/go-gtp/gtpv2/message"
+	"github.com/wmnsk/go-gtp/gtpv2/testutils"
+)
+
+func TestChangeNotificationResponse(t *testing.T) {
+	cases := []testutils.TestCase{
+		{
+			Description: "Normal",
+			Structured: message.NewChangeNotificationResponse(
+				testutils.TestBearerInfo.TEID, testutils.TestBearerInfo.Seq,
+				ie.NewIMSI("123451234567890"),
+				ie.NewMobileEquipmentIdentity("123450123456789"),
+				ie.NewPrivateExtension(10415, []byte{0xde, 0xad, 0xbe, 0xef}),
+			),
```

Comment:
Is it possible to cover all IEs?

Reply:
When analyzing test scripts, these fields were never checked. Not sure if this is necessary within the current task.